microsoft/testsuites/performance/common.py

```diff
@@ -40,10 +40,10 @@ def perf_disk(
     start_iodepth: int,
     max_iodepth: int,
     filename: str,
-    core_count: int,
-    disk_count: int,
-    disk_setup_type: DiskSetupType,
-    disk_type: DiskType,
+    core_count: Optional[int],
```

Comment:
What's the reason to mark them optional? In the new change, they are assigned.

Reply:
Yes in latest push we assigned core_count as per core and disk_count as 1 by default. We need to make disk_setup_type and disk_type as option so we can pass None to them and there will be minimal change in signature of perf_disk as new testcase does not require disk specification. This is generic FIO test which can run as per passed params in runbook.

I will remove optional from core_count  and disk_count